contrib/python/podman/podman/libs/containers.py

```diff
@@ -1,38 +1,40 @@
 """Models for manipulating containers and storage."""
 import collections
-import functools
 import getpass
 import json
 import logging
 import signal
 import time
 
+from . import fold_keys
 from ._containers_attach import Mixin as AttachMixin
 from ._containers_start import Mixin as StartMixin
 
 
 class Container(AttachMixin, StartMixin, collections.UserDict):
     """Model for a container."""
 
-    def __init__(self, client, id, data):
+    def __init__(self, client, ident, data, refresh=True):
         """Construct Container Model."""
         super(Container, self).__init__(data)
-
         self._client = client
-        self._id = id
+        self._id = ident
 
-        with client() as podman:
-            self._refresh(podman)
+        if refresh:
+            with client() as podman:
+                self._refresh(podman)
+        else:
+            for k, v in self.data.items():
+                setattr(self, k, v)
+            if 'containerrunning' in self.data:
+                setattr(self, 'running', self.data['containerrunning'])
+                self.data['running'] = self.data['containerrunning']
```

Comment:
(maybe I am dumb)

Why the ``setattr(self, ...)`` and not just ``self.running = ...``?  Are there side-effects?  Might be helpful to future developers to add a comment about this.  Then the confused look on their face won't persist :grinning:

Reply:
@cevich Habit, since I just did a setattr() above a C&P'ed it down for the alias. But given it's a known string you are right that code be just an "assignment".